### PR TITLE
Issue 7/set local storage as memory method

### PR DESCRIPTION
**Pull request by jawiemkto**

### CODE REVIEW

File: src/core/Memory.ts [hunk 1]
```diff
@@ -1,9 +1,16 @@
-let values_in_browser: { date: Date, value: number }[] = [];
+export const storageKey = 'values';
+
+function parseValueDateToBeDateObj (valuesFromLS: { date: string, value: number }[]): { date: Date, value: number }[] {
+    return valuesFromLS.map(dataObj => ({
```
Comment (oneacik) [inline src/core/Memory.ts]: I come from Java environment where we are not scared of long names.
Here using valuesFromLocalStorage sound better.

Also I would name your function like recoverTypesFromParsedJson to make it more understandable what is it doing.

File: src/core/Memory.test.ts [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
 test("Test That Value Retrieved will Be Empty If not Set", () => {
     expect(getValues()).toEqual([]);
 })
 
-test("Test that Value Can Be Saved And Retrieved", () => {
-    const value = [{date: new Date(), value: 21}, {date: new Date(), value: 48}];
-    saveValues(value);
-    expect(getValues()).toEqual(value);
+test('When saveValues is executed Then data should be saved in localstorage', () => {
+    saveValues(valuesArrMock);
```
Comment (oneacik) [inline src/core/Memory.test.ts]: short names XD

btw. I think that a better name is CORRECT_VALUE_TO_SAVE.
SNAKE_CASE is for constants.

**Review by oneacik — APPROVED** [review]
I'm giving approve, just change those names maybe <- optional

File: src/core/Memory.ts (hunk 1 above)
Comment (jawiemkto) [reply src/core/Memory.ts]: :) agree

**Pull request merged**